modules/tide_edit_protection/tide_edit_protection.install

```diff
@@ -12,7 +12,7 @@ use Drupal\views\Entity\View;
  */
 function tide_edit_protection_install() {
   module_load_include('inc', 'tide_core', 'includes/helpers');
-  $config_location = [drupal_get_path('module', 'tide_edit_protection') . '/config/install'];
+  $config_location = [drupal_get_path('module', 'tide_edit_protection') . '/config/optional'];
```

Comment:
Why is it changing to optional? It's a hard dependency isn't it?

Reply:
Hi @anthony-malkoun , 
Thanks.
I would like the `views.view.tide_locked_content` view to be installed at very last step.
I also made another update, so that the installation steps will be :
content_lock -> content_lock_timeout -> tide_edit_protection -> views.view.tide_locked_content